hack/buildkitd_build_push_image.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+HIVE_ROOT="$(git rev-parse --show-toplevel)"
+CNI_PATH="$HIVE_ROOT"/.tmp/cni/bin
+export HIVE_ROOT
+export CNI_PATH
+export PATH=$HIVE_ROOT/.tmp/_output/bin:$PATH
+
+# Run BuildKit in the background
+echo "Starting BuildKit daemon in the background..."
+rootlesskit buildkitd > /dev/null 2>&1 &
```

Comment:
https://github.com/moby/buildkit/blob/3a7612783149d8ca023fab7b2875de1b8d3e5071/docs/rootless.md#running-buildkit-in-rootless-mode-containerd-worker

Maybe this simplifies the shutdown of buildkitd